src/app.js

```diff
@@ -45,7 +45,8 @@ import { registerHandlers } from './handlers/index';
 
 import OnboardingBox from './components/onboardingBox';
 
-const FB_POST_SELECTOR = '[role="article"]';
+// const FB_POST_SELECTOR = '[role="article"]';
+export const FB_POST_SELECTOR = '.fbUserContent';
```

Comment:
I'll add it, but in the past, it was `.fbUserContent` or `.userContentWrapper` or `.fbContentWrapper`, I guess we've to address the issue in some other way...